assets/js/functions.js

```diff
@@ -18,3 +18,46 @@ function loadFaqComponent() {
         }
     );
 }
+
+
+//function to load blog posts
+function loadBlogPosts() {
+    $.ajax({
+        type: 'get',
+        url: 'https://script.google.com/macros/s/AKfycbyUdfhk9WWFLoMWxQvNIEUNl793RbXGnyo-ZIYRn9K1T6QFN1_9G8Ln9NpTCCJba2dzOg/exec',
+        dataType: 'json',
+        success: function (data) {
+            document.getElementById('btnBlogPosts').style.visibility = "visible";
+            //slice array to two parts
+            if (data.length >= 6) {
+                let partOne = data.slice(0, 6);
+                let partTwo = data.slice(6);
+
+                //mustache render - part one
+                let contentPartOne = Mustache.render(
+                    $('#template-blog-posts').html(), { 'data': partOne });
+
+                //display first 8 profiles
+                $('#blog-posts').html(contentPartOne);
+
+                //hide button
+                $('#btn-blog-posts-show-more').click(function () {
+                    let contentPartTwo = Mustache.render(
+                        $('#template-blog-posts').html(), { 'data': partTwo });
+                    $(contentPartTwo).appendTo('#blog-posts').hide().fadeIn(1000);
+                    $('#btn-blog-posts-show-more').hide();
+                });
+            } else {
+                //mustache render
+                let content = Mustache.render(
+                    $('#template-blog-posts').html(), { 'data': data });
+
+                //display first 8 profiles
+                $('#blog-posts').html(content);
+
+                //hide button
+                $('#btn-blog-posts-show-more').hide();
+            }
+        }
+    });
+}
```

Comment:
We have this function in a couple of places. Shall we make this a generalized function where are pass the parameters?

Reply:
The `scholarx/2022/functions.js` and `assets/js/functions.js` files both include the same function. Therefore, shall I write a parameterized function in `assets/js/functions.js` and use it in both places?

Reply:
can you check the past scholarx pages as well? It should be there as well.

Reply:
It's used on the 2020 & 2021 pages as well

Reply:
I'll write the function in the file `assets/js/functions.js` and use it in those files. The CSS will also be removed from the other locations and moved to the `assets/css/global-custom.css` file. That's ok, Right?

Reply:
yeah that's better

Reply:
@anjula-sack  There is a difference in the 2021 page cards, it includes a border radius. Should we remove it from the 2021 page or should we add it to the other pages as well?

**2021 page**
![Screenshot from 2022-12-02 10-40-47](https://user-images.githubusercontent.com/63200586/205219639-bc80b5b2-ccfd-4192-8d23-b4a1ac78451a.png)

**Other pages**
![Screenshot from 2022-12-02 10-41-05](https://user-images.githubusercontent.com/63200586/205219631-853b7d08-c68c-48b9-8524-3118624540d7.png)

Reply:
There is a height difference as well